dali/test/python/test_pipeline.py

```diff
@@ -1619,9 +1622,20 @@ def test_output_dtype_deprecation():
         # Verify DeprecationWarning
         assert len(w) == 1
         assert issubclass(w[-1].category, DeprecationWarning)
-        assert ("Argument name 'output_dtype' for operator CropMirrorNormalize is deprecated. " +
-                "Use 'dtype' instead.") == str(w[-1].message)
+        assert ('Argument "output_dtype" for operator "CropMirrorNormalize" is now deprecated. ' +
```

Comment:
I used the `'` as this is what Python is using, but I don't now how it goes with rest of DALI.

There is image_type specified and it was also deprecated, so this test will fail I think.